sdk/monitor/azure-monitor-opentelemetry-exporter/README.md

```diff
@@ -1,6 +1,8 @@
 # Microsoft OpenTelemetry exporter for Azure Monitor
 
-The exporter for Azure Monitor allows you to export data utilizing the OpenTelemetry SDK and send telemetry data to Azure Monitor for applications written in Python.
+The exporter for Azure Monitor allows Python applications to export data from the OpenTelemetry SDK to Azure Monitor. The exporter is intended for users who require advanced configuration or has more complicated telemetry needs that require all of distributed tracing, logging and metrics. If you have simpler configuration requirements, we recommend using the [Azure Monitor OpenTelemetry Distro](https://learn.microsoft.com/azure/azure-monitor/app/opentelemetry-enable?tabs=python) instead for a simpler one-line setup.
```

Comment:
"...The exporter is intended for users who require advanced configuration or have* more complicated telemetry needs.*" Pluralize a verb + realizing that this makes it sound like the distro doesn't include tracing, logging, and metrics.